src/libopensc/card-jpki.c

```diff
@@ -265,6 +271,7 @@ jpki_pin_cmd(sc_card_t * card, struct sc_pin_cmd_data *data, int *tries_left)
 		if (tries_left) {
 			*tries_left = apdu.sw2 - 0xC0;
 		}
+		data->pin1.logged_in = priv->state;
```

Comment:
No JPKI card always return tries_left when calling "00 20 00 80"
There seems to be only same way with PIV.

~~~
> 00 20 00 80 04 XX XX XX XX # correct PIN
<  90 00
> 00 20 00 80
<  63 C3
~~~

Well, I can't found description around login state in ISO-7816 spec.
http://www.cardwerk.com/smartcards/smartcard_standard_ISO7816-4_6_basic_interindustry_commands.aspx#chap6_12